
std_msgs/Header header
FingerState[3] finger_state
float64 preshape_angle